README.md

```diff
@@ -154,6 +154,10 @@ To build the project, run `npm start` in the root directory. (If you're running
 To test it out, use the `/example` directory:
 
 ```
+# from the root
+npm install
+npm start
```

Comment:
add install instructions

- keep npm start running on root repo to bundle changes into /dist for the example repo